tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature

```diff
@@ -152,3 +152,18 @@ Feature: users management
       | Brian |
       | Carol |
     And "Admin" logs out
+
+
+  Scenario: edit panel can be opened via quick action and context menu
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+      | Carol |
+    And "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the users management page
+    When "Admin" opens the edit panel of user "Brian" using the quick action
+    Then "Admin" should see the edit panel
+    When "Admin" opens the edit panel of user "Brian" using the context menu
+    Then "Admin" should see the edit panel
```

Comment:
duplicate actions

Reply:
no actually the tests is to check if the edit panel can be opened using the `quick action` and `context menu`

Reply:
yes, sorry